src/vivarium_census_prl_synth_pop/constants/metadata.py

```diff
@@ -295,3 +295,6 @@ class DatasetNames:
     "date_of_birth": "copy_date_of_birth",
     "has_ssn": "copy_ssn",
 }
+
+
+YEAR_AGGREGATION_VALUE = 3000
```

Comment:
What is this?

Reply:
Maybe -1 would be better than?

Reply:
Also maybe this variable should be changed to make it clear that this is a nonsense value.